### PR TITLE
Section 7 updates

### LINKED ISSUE #169: all AIA URLs in Subordinate CA certs should be publicly accessible
In 7.1.2.2 c change "At least one instance" to "All instances"
Having one or more URLs in public certificates not be resolvable can create timing issues for RP applications trying to do dynamic path discovery & validation

### LINKED ISSUE #185: EKU on subordinate CA certs
line 2007 - add and may include the id‐kp‐clientAuth extended key usage
Make the statement about name Constraints a separate sentence.

### LINKED ISSUE #433: Section 7
**Organization / Program:** DoD <br>**Section:** 7 <br>**PDF Page:** N/A<br>**PDF Line(s):** N/A<br>**Comment:** My general recommendation is to delete all the material from this section that is covered by the profiles.  Thus, very little should remain.  This will help reduce the size of the CP and will eliminate the need to keep the CP and profiles consistent.  I have made many comments on the profiles.  While going through the CP I may miss some of those comments, so set theoretic union of these and profile comments should be use to revise the profiles and this section<br><br>**Suggested Change:** None

### LINKED ISSUE #435: Section 7.1.2.2
**Organization / Program:** DoD <br>**Section:** 7.1.2.2 <br>**PDF Page:** 70<br>**PDF Line(s):** 1840<br>**Comment:** See comment on the profiles.  Why can path length of 0 or 1 be asserted depending on Intermediate or Sub CA?<br><br>**Suggested Change:** None

### LINKED ISSUE #436: Section 7.1.2.3
**Organization / Program:** DoD <br>**Section:** 7.1.2.3 <br>**PDF Page:** 71<br>**PDF Line(s):** 1866<br>**Comment:** Note that test certificates call for this to be critical.  "SHOULD" is duly noted.  Nonetheless, some clarifying text if critical extension for test certificate is asserted, is warranted.<br><br>**Suggested Change:** None

### LINKED ISSUE #438: Section 7.1.3
**Organization / Program:** NSA <br>**Section:** 7.1.3 <br>**PDF Page:** 72<br>**PDF Line(s):** 1921<br>**Comment:** There are no algorithm OIDs specified here<br><br>**Suggested Change:** specify the OIDs for the algorithms listed in Section 6.1.5.

### LINKED ISSUE #439: Section 7.1.3
**Organization / Program:** DoD <br>**Section:** 7.1.3 <br>**PDF Page:** 72<br>**PDF Line(s):** 1922<br>**Comment:** Simply point to profiles for crypto algorithms<br><br>**Suggested Change:** Replace text with "Certificate Profiles describe algorithms used in signing certificates and algorithms for the subject public key information."

### LINKED ISSUE #440: Section 7.1.5
**Organization / Program:** DoD <br>**Section:** 7.1.5 <br>**PDF Page:** 75<br>**PDF Line(s):** 2025<br>**Comment:** All issuer and subject DN are directory names.<br><br>**Suggested Change:** Delete the line

### LINKED ISSUE #443: Section 7.3.2
**Organization / Program:** NSA <br>**Section:** 7.3.2 <br>**PDF Page:** 78<br>**PDF Line(s):** 2113<br>**Comment:** Where is the OCSP no check extension specified?<br><br>**Suggested Change:** None

### LINKED ISSUE #444: Section 7.3.2
**Organization / Program:** DoD <br>**Section:** 7.3.2 <br>**PDF Page:** 78<br>**PDF Line(s):** 2132-2136<br>**Comment:** With no check OCSP field does not make sense.  Since you already have CA certificate for the subject from the certification path, CA issuer field is also not necessary.<br><br>**Suggested Change:** Align with recommendation on profiles making this field optional and containing only CA Issuers method.

### LINKED ISSUE #445: Section 7.3.2.b
**Organization / Program:** NSA <br>**Section:** 7.3.2.b <br>**PDF Page:** 78<br>**PDF Line(s):** 2135<br>**Comment:** What is the point of putting in the OCSP responder URL in the OCSP certificate?<br><br>**Suggested Change:** None

### LINKED ISSUE #470: Section 7 Editorial
Additional comments received on Section 7

| Organization / Program | PDF Line | Comment | Change |
| --- | --- | --- | --- |
| DoD|1855-1857|In order to enure that clients that enforce EKU on path, add the OCSP Signing to the list of mandatory EKU|Add OCSP Signing to the list of mandatory EKUs |
| DoD|1908-1915|We are not using any of this|Delete this text |
| DoD|1983-1984|It is not a device PKI.  It is a TLS Server PKI|Here and throughout the CP replace the notion of device with TLS Server |

### PR DESCRIPTION
**Pull request by lachellel**
- Moving most of existing section 7 out and referencing the profiles
- Profiles appended as Appendix D using _includes_
- One more set of intros for each profile will be needed (not too wordy trust me) but the teams need the section 7 / profiles updates asap for the test environments
 

closes #470 closes #445 closes #444 closes #443 closes #440 closes #439 closes #438 closes #436 closes #435 closes #433 closes #169 closes #185

### CODE REVIEW

**Comment by lachellel** [comment]
@techliaison @konklone 

take a look please?  note my note on adding a very small intro to each profile if it's **really needed**.

File: certificate-policy.md [hunk 1]
```diff
@@ -496,15 +496,8 @@ All Domain Validation TLS Server Authentication certificates and Organizational
 
 There is no limit on the maximum number of CT Logs which may be submitted to.  
 
-The Issuing CA shall include at least the same number and variety of SCTs in the x509v3 certificate extension for the Domain Validation TLS Server Authentication certificate or the Organizational Validation TLS Server Authentication certificate issued.
-<<<<<<< HEAD
-<<<<<<< 971515f870f94edf1a6c16f3de4605cfd0068a70
-=======
+The CA shall include at least the same number and variety of SCTs in the x509v3 certificate extension for the Domain Validation TLS Server Authentication certificate or the Organizational Validation TLS Server Authentication certificate issued.
```
Comment (techliaison) [inline certificate-policy.md]: I still think the "at least the same number" is confusing because this sentence comes after "there is no limit on the maximum number of CT logs which may be submitted to"
can we move that sentence to after this one?

**Comment by lachellel** [comment]
@konklone - wendy reviewed and i included her updates

I'm going to merge to start the last round again (another round of comments in from one of the teams).
